prow/spyglass/lenses/junit/template.html

```diff
@@ -21,38 +22,127 @@
     </tr>
     <tbody id="failed-tbody">
     {{range $ix, $test := .Failed}}
+      {{$numTest := len $test.Junit}}
+      {{$firstTest := index $test.Junit 0}}
+      {{if eq $numTest 1}}
+        <tr>
+          <td colspan="2" style="padding: 0;">
+            <table class="failed-layout">
+              <tr class="failure-name">
+                <td class="mdl-data-table__cell--non-numeric test-name">{{$firstTest.Name}}&nbsp;<i class="icon-button material-icons arrow-icon">expand_more</i></td>
+                <td class="mdl-data-table__cell--non-numeric" style="text-align: right;">{{$firstTest.Duration}}</td>
+              </tr>
+              <tr class="hidden failure-text">
+                <td colspan="2" class="mdl-data-table__cell--non-numeric">
+                  <div>{{$firstTest.Failure}}</div>
+                  {{if $firstTest.Output}}
+                  <a href="#" class="open-stdout">open stdout<i class="material-icons" style="font-size: 1em; vertical-align: middle; padding-left: 3px;">open_in_new</i></a>
+                  <pre style="display: none;">{{$firstTest.Output}}</pre>
+                  {{end}}
+                </td>
+              </tr>
+            </table>
+          </td>
+        </tr>
+      {{else}}
+      <tr>
```

Comment:
nit: This block should be at the same indentation level as in the 'if' above. (same with other blocks under if/else controls, for consistency).

Reply:
done with indentation